modulefiles/gsi_hera.gnu.lua

```diff
@@ -1,15 +1,15 @@
 help([[
 ]])
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.5.1/envs/gsi-addon/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/gsi-addon-dev/install/modulefiles/Core")
 --Needed for openmpi build
 prepend_path("MODULEPATH", "/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles")
 
-local python_ver=os.getenv("python_ver") or "3.10.8"
+local python_ver=os.getenv("python_ver") or "3.11.6"
 local stack_gnu_ver=os.getenv("stack_gnu_ver") or "9.2.0"
 local stack_openmpi_ver=os.getenv("stack_openmpi_ver") or "4.1.5"
 local cmake_ver=os.getenv("cmake_ver") or "3.23.1"
-local prod_util_ver=os.getenv("prod_util_ver") or "1.2.2"
+local prod_util_ver=os.getenv("prod_util_ver") or "2.1.1"
 local openblas_ver=os.getenv("openblas_ver") or "0.3.19"
```

Comment:
I believe openblas version should be 0.3.24, @DavidHuber-NOAA can you confirm?

Reply:
@AlexanderRichert-NOAA You are correct, it should be 0.3.24.  I updated it and ran a test build.  Thanks!